src/components/ContactInfo.js

```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const ContactInfo = props => (
```

Comment:
not necessary component this could be done in ContactItem

Reply:
Moved to ContactItem.